model/Core/Classes/PackageVerificationCode.md

```diff
@@ -18,19 +18,21 @@ This identifier enables a recipient to determine if any file in the original pac
 
 Algorithm:
 
+    hashValue = 0
```

Comment:
Why setting it to an initial value, if you plan to overwrite it with an assignment at the end? This is never used.

Reply:
A hash of a null string gives unpredictable valued, based on implementation environement. 

I'll recode this though to be clearer with an "if" at the end.